features/utils/pageobjects/home.page.js

```diff
@@ -0,0 +1,57 @@
+import Element from '../elements/base.element';
+import Link from '../elements/link';
+import Input from '../elements/input';
+import DropDown from '../elements/drop.down';
+import Button from '../elements/button';
+
+class HomePage {
+  get toursTab() {
+    return new Link('a.tours');
+  }
+
+  get inputTour() {
+    return new Input('.locationlisttours >input');
+  }
+
+  get tourInDropDown() {
+    return new DropDown('.select2-results-dept-1 >.select2-result-label >.select2-match');
+  }
+
+  get tourType() {
+    return new Element('#tourtype_chosen');
+  }
+
+  get tourTypeInDropDown() {
+    return new DropDown('[data-option-array-index="4"]');
+  }
+
+  get inputForDateTour() {
+    return new Input('#tours #DateTours');
+  }
+
+  get buttonForIncreaseAmountOfAdults() {
+    return new Button('#tours input[name="adults"] ~ .input-group-btn-vertical > button[class*="bootstrap-touchspin-up"]');
```

Comment:
Consider passing the general rule for all strings passing to the selectors :
 get buttonForIncreaseAmountOfAdults() { 
return new Button('#tours input[name="adults"] ~ .input-group-btn-vertical > button[class*=
**"**bootstrap-touchspin-up**"** 
in this case and   
get inputEmail() {
    return new Input('.form-group > [name =**\'**email**\'**]');
  }
in other case